packages/core/src/error_handler.ts

```diff
@@ -59,12 +59,11 @@ export class ErrorHandler {
 
   /** @internal */
   _findContext(error: any): any {
-    if (error) {
-      return getDebugContext(error) ? getDebugContext(error) :
-                                      this._findContext(getOriginalError(error));
+    if (!error) {
+      return null;
     }
 
-    return null;
+    return getDebugContext(error) || this._findContext(getOriginalError(error));
```

Comment:
If we wanted we could even avoid the `return null` and make this a one liner (not sure if it's less readable)

Also I think that maybe the nullish coalescing operator here would be more appropriate than the or, wouldn't it?

```suggestion
    return !error ? null : getDebugContext(error) ?? this._findContext(getOriginalError(error));
```

Reply:
thanks for your suggestion. please review again.